Sabetskyi_Stanislav/src/TelephoneCall.java

```diff
@@ -0,0 +1,24 @@
+import java.util.Scanner;
+public class TelephoneCall {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        System.out.print("Input 5 first numbers of your phone number:");
+        int phone_number = in.nextInt();
+        double CostOfCall = 0;
+        String CityName = null;
+        switch (phone_number) {
+            case 38044: CityName="Kiev";  CostOfCall= 1.25;
```

Comment:
Don't forget using `ctrl + alt + L` before each time you commit code 😉

Reply:
OK